erpnext_germany/erpnext_germany/doctype/business_trip/business_trip.py

```diff
@@ -212,3 +212,44 @@ def get_meal_expenses(business_trip: BusinessTrip, expense_claim_type: str) -> l
 		)
 
 	return expenses
+
+
+@frappe.whitelist()
+def get_processing_details(business_trip: str):
+	"""Get linked Expense Claims and Purchase Invoices for the Business Trip"""
+	frappe.has_permission(doctype="Business Trip", doc=business_trip, ptype="read", throw=True)
```

Comment:
```suggestion
	frappe.has_permission("Business Trip", doc=business_trip, throw=True)
```

`"read"` ist der default-Wert für `ptype` und muss nicht explizit angegeben werden. Der DocType-Parameter am Anfang wird meistens nicht benannt (übliches Muster in der Frappe-Welt).